deployment/nginx.coffee

```diff
@@ -460,3 +462,18 @@ createHttpsRedirector = (KONFIG) ->
       # use generic names, do not hardcode values
       return 301 https://$host$request_uri;
     }"""
+
+createHttpOptions = (KONFIG) ->
```

Comment:
Could you combine this  and createListenDirective together to have a unified functionality? (we can set the defaults for options.nginx for createListenDirective )

Reply:
real_ip module directives are set in `http` block. It applies to everywhere functionally. Is this what you are suggesting?

Reply:
There is a function in this file called "createListenDirective" which has pretty similar body.